pkg/clients/kubernetes.go

```diff
@@ -9,6 +9,24 @@ import (
 	"k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
 )
 
+// Kubernetes - Create a new kubernetes client if needed, returns reference
+func Kubernetes(log *logging.Logger) *clientset.Clientset {
```

Comment:
The only other thing that I might want to change is to add an error to this call. 
I could see a time when we might be needing a pool, and we would use this method to retrieve values and would want errors. I also think that making sure the value is not nil, would be something that could be useful right now.
Anyone have strong feelings this is just an idea I don't fee strongly about it.